logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/mlsmigration/MLSMigrationManager.kt

```diff
@@ -90,12 +89,16 @@ internal class MLSMigrationManagerImpl(
         ).flatMap { lastMlsMigrationCheckHasPassed ->
             kaliumLogger.d("Migration needs to be updated: $lastMlsMigrationCheckHasPassed")
             if (lastMlsMigrationCheckHasPassed) {
-                kaliumLogger.d("Updating migration config")
-                mlsMigrationRepository.value.fetchMigrationConfiguration()
-                    .onSuccess { timestampKeyRepository.value.reset(TimestampKeys.LAST_MLS_MIGRATION_CHECK) }
-                    .flatMap {
-                        mlsMigrationWorker.value.runMigration()
-                        Either.Right(Unit)
+                kaliumLogger.d("Running mls migration")
+                println("running migration")
```

Comment:
question: missed debug logs? or if it's needed convert to logger?

Reply:
I'll delete them